node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/FailedExpirer.java

```diff
@@ -127,11 +127,4 @@ private boolean failCountIndicatesHardwareIssue(Node node) {
         return (zone.environment() == Environment.prod || zone.environment() == Environment.staging) &&
                node.status().failCount() >= maxAllowedFailures;
     }
-
-    /** Returns whether node has any kind of hardware issue */
-    private static boolean hasHardwareIssue(Node node) {
```

Comment:
Not used elsewhere. Moved to NodeFailer because it is responsible for what hardware issue really mean. Unfortunately it needs a NodeRepository to look up the parent (to figure out whether the host has reports), so I don't see how it would fit into a NodeList.